quickstarts/aws/aws-athena/config.yml

```diff
@@ -1,7 +1,7 @@
 id: 5f3d35d7-668b-41f3-98d7-ccc93a50933c
-slug: amazon-athena
+slug: aws-athena
```

Comment:
In general, we do not change the slug on quickstarts because that is used as part of the url pathing to a particular quickstart. If we really need this change, let's sync on ways to do this without breaking links for people.

Reply:
I'll change the slug back and I've let everyone know to not change that. 
I'll also try to find accounts with data for `dashboards/aws-appsync` and `dashboards/aws-athena` though regarding `dashboards/aws-auto-scaling`, the screenshots do seem to have data in them, can you confirm?

Reply:
Unfortunately for `dashboards/aws-appsync` and `dashboards/aws-athena`, I've not been able to find any non-0 data.